src/cargo/core/manifest.rs

```diff
@@ -204,6 +207,7 @@ impl Manifest {
     pub fn version(&self) -> &Version { self.package_id().version() }
     pub fn warnings(&self) -> &[String] { &self.warnings }
     pub fn profiles(&self) -> &Profiles { &self.profiles }
+    pub fn private(&self) -> &bool { &self.private }
```

Comment:
It's ok for this to just return a `bool` instead of `&bool` because it's a `Copy` type